configure.ac

```diff
@@ -67,6 +67,12 @@ PKG_CHECK_MODULES([XF86VM], [xxf86vm], [have_xf86vm=yes], [have_xf86vm=no])
 PKG_CHECK_MODULES([XCB], [xcb], [have_xcb=yes], [have_xcb=no])
 PKG_CHECK_MODULES([XCB_RANDR], [xcb-randr],
 	[have_xcb_randr=yes], [have_xcb_randr=no])
+PKG_CHECK_MODULES([WAYLAND], [wayland-client wayland-scanner], [have_wayland=yes], [have_wayland=no])
+AC_PATH_PROG([wayland_scanner], [wayland-scanner])
```

Comment:
These 5 lines appear to be causing the build to fail on Travis, which I think is legitimate. The configuration script should only look for `WAYLAND_SCANNER` if Wayland is available. It is not doing that at present, but I'm not familiar enough with `autoconf` syntax to know precisely why.

Reply:
I fixed this in https://github.com/breznak/redshift/pull/3

Reply:
Thanks for finding the culpit and fixing that, guys! Merged the update and triggering a new CI run here...